travel/Html/travel_platform.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
```

Comment:
We don't need this file in the project. I'm going to delete testing folder before merging.